README.md

```diff
@@ -1,8 +1,12 @@
 # h26x-extractor
 
+## Why the fork?
+
+When I've started working on this, I disregarded whether the original author would or would not accept pull requests. So, this was why.
+
```

Comment:
Please undo this part so I can merge into my repo.

Reply:
Rolled back.